### ISSUE TITLE
MHV datafeed for Option 2 Sign In and all total MHV credentials

### ISSUE THREAD

**Issue by djmunoz1** [department-of-veterans-affairs/va.gov-team#64246]

**Comment by lmorris3** [comment]
@sdickson-gt is this work associated with the data cleanup you recently completed, do you know?

**Comment by sdickson-gt** [comment]
No. We are currently getting authentication data for SSOe and SiS, but we aren't accounting for MHV direct authentications via MHV Option 2 in our monthly authentication by CSP reporting.
